libpod/container_internal_linux.go

```diff
@@ -2367,3 +2378,37 @@ func (c *Container) checkFileExistsInRootfs(file string) (bool, error) {
 	}
 	return true, nil
 }
+
+// getSecret gets a secret to mount into the container
```

Comment:
What are the return values?

Reply:
Although it might seem strange at first glance, I prefer using more specific function names. `getObject` is too generic and I wouldn't guess that files are being created.

Maybe we can name it `extractSecretAndPrepareMount`?  Just an example for a more expressive function name.

Reply:
We also need to handle errors better here. If a secret a container is using is removed, the container becomes unusable. Is that desirable, or should we proceed with a warning?

Reply:
Hmm I have to take another look at what I'm doing, in a create/start vs a run.